lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -22,10 +22,10 @@ const UISTRINGS_REGEX = /UIStrings = (.|\s)*?\};\n/im;
  */
 
 const ignoredPathComponents = [
-  '/.git',
-  '/scripts',
-  '/node_modules',
-  '/test/',
+  `${path.sep}.git`,
+  `${path.sep}scripts`,
+  `${path.sep}node_modules`,
+  `${path.sep}test${path.sep}`,
```

Comment:
i kinda hate this, wb this instead

```
[
  '/.git',
  '/scripts',
  '/node_modules',
  '/test/',
  '-test.js',
  '-renderer.js',
].map(str => str.replace(/\//g, path.sep));
```

Reply:
Meh, this seems even more obtuse imo.

Reply:
I actually like that @connorjclark, the readability of the paths is the important thing IMO, the fact that we want to support multiple OS is the secondary concern, maybe just add the 

`// Ensure we use the OS-specific path separator` comment to the map line?